docs/modules/ROOT/pages/how-to/use-vpa.adoc

```diff
@@ -0,0 +1,80 @@
+= How to use Vertical Pod Autoscaler to adjust Pod Resouce levels
+
+It's complicated to find out good values for xref:how-to/check-cpu-requests.adoc[investigate and resolve fair use ratio violations for a project]. APPUiO Cloud offers the OpenShift https://docs.openshift.com/container-platform/4.11/nodes/pods/nodes-pods-vertical-autoscaler.html[Vertical Pod Autoscaler (VPA)] object to assist DevOps engineers to fine tune the requests and limits of their deployments.
+
+This document explains how to use VPAs. This document assumes that you have a working project with some payload deployed and running.
+
+== Limitations
+
+The VPA operator is provided in APPUiO Cloud with the following limitations:
+
+* The VPA operator only provides recommendations; it doesn't recreate pods automatically with new request and limit values.
+
+== Create the VPA Object
+
+Use the YAML below to define a new VPA object, and `oc apply` it to your namespace.
+
+[source,yaml]
+--
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: vpa-recommender
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind:       Deployment
+    name:       vpa-example  # <1>
+  updatePolicy:
+    updateMode: "Auto"
```

Comment:
I'd use `updateMode: Off` to ensure that existing configurations don't start recreating pods if we decide to deploy the `vpa-updater` as well at some point.

```suggestion
    updateMode: "Off"
```